### PR TITLE
Fix Marker Component

### PR DESCRIPTION
**Pull request by basantech89**
## Details

Fix: #406 

Fix Marker Component
Below is the screenshot:-
2020-01-21-1579585934_screenshot_1366x768

**Completed?:** (Yes)

### CODE REVIEW

**Comment by NickFoden** [comment]
Wait @basantech89  @auser  I think this might be a mistake. 

This returns null instead of a marker ? 

image

**Comment by auser** [comment]
Oops! I may have included that when I intended on commenting instead.

File: src/components/Marker.js [hunk 1]
```diff
@@ -104,20 +104,7 @@ export class Marker extends React.Component {
...
-              this.props.children, 
-              { marker: this.marker,
-                google: this.props.google,
-                map: this.props.map}
-            )
-          ) : null
-        }
-      </Fragment>
-    )
+    return null;
```
Comment (auser) [inline src/components/Marker.js]: Why are we removing this entirely?

**Comment by NickFoden** [comment]
Yeah @auser I think revert this. OP may need to pass the markers in a different format for their needs. 

@basantech89 this is how i render markers make sure you have all the required props for your markers that could be the reason you don't have working markers. Ping me with your code if you are unable to render markers on your map. 

```    
this.props.locationMarkers.map((item, index) => {
return (
<Marker
   key={index}
   id={index}
   icon={MapMarker}
   position={{
      lat: item.lat,
      lng: item.lng
      }}
    name={item.address ? item.address : ""}
    url={`rental-listing/${item.url}`}
    onClick={this.onMarkerClick}
/>
```

**Pull request merged**